Demo/Demo/Forms/FormsListViewController.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  FormsListViewController.swift
+//  Demo
+//
+//  Created by Toni Kocjan on 29/07/2021.
+//
+
+import UIKit
+import PovioKit
+
+class FormsListViewController: UIViewController {
+  let tableView = UITableView()
+  let validationForms: [(name: String, form: ValidationForm)]
+  
+  init(validationForms: [(name: String, form: ValidationForm)]) {
+    self.validationForms = validationForms
+    super.init(nibName: nil, bundle: nil)
+    setup()
+  }
+  
+  required init?(coder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+  
+  override func loadView() {
+    view = tableView
+  }
+}
+
+extension FormsListViewController: UITableViewDataSource {
+  func numberOfSections(in tableView: UITableView) -> Int {
+    1
+  }
+  
+  func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+    validationForms.count
+  }
+  
+  func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+    let form = validationForms[indexPath.row]
+    let cell = UITableViewCell(style: .value1, reuseIdentifier: "cell")
+    cell.textLabel?.text = form.name
+    return cell
+  }
+}
+
+extension FormsListViewController: UITableViewDelegate {
+  func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+    let form = validationForms[indexPath.row].form
+    let vc = FormViewController(form: form)
```

Comment:
As mentioned above, not sure about this, would prefer to  keep the current structure we use and have the form "embeded" into ViewController's `contentView` and not need to inject it.

Reply:
This is just a demo showcasing how the validation from builder works. The way it is used in the demo is not meant to be translated into a real world project.